index.html

```diff
@@ -0,0 +1,50 @@
+<html>
+    <head>
+        <title>My Simple Base Page</title>
+        <link rel="stylesheet" type="text/css" href="style.css"/>
+        <link href="https://fonts.googleapis.com/css?family=Allan|Asap+Condensed&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <header class="navigation" >
+                <button id="burgerBtn"><i class="fa fa-bars"></i></button>
+                <nav> <a href >Home</a></nav>
+               <nav> <a href>About me</a></nav>
+               <nav class="base"> <a href="#index.html">Base Page</a></nav>
+            
```

Comment:
not needed empty line. And below your using `<br/>` which is not needed. You should be able to achieve the same with css styles.